File-Formats/Presentation/Presentation-to-image.md

```diff
@@ -615,11 +619,11 @@ using (FileStream fileStreamInput = new FileStream(@"Template.pptx", FileMode.Op
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PPTX-to-Image-conversion/Customize-default-fallback-fonts).
+### Fallback fonts for range of Unicode text
 
-### Add Custom Fallback Fonts
+Users can set fallback fonts for specific Unicode range of text to be used in presentation to Image conversion.
 
-The following code example demonstrates how to add custom fallback fonts while converting a PowerPoint presentation to Image.
+The following code example demonstrates how users can add fallback fonts by using a specific Unicode range of text that Presentation considers internally while converting a PowerPoint presentation to Image.
```

Comment:
The following code example demonstrates how users can add fallback fonts by using a specific Unicode range of text that Presentation considers internally while converting a PowerPoint presentation to an Image.

Reply:
Modified the sentences